CONTRIBUTING.md

```diff
@@ -40,11 +46,10 @@ AUTH0_CLIENT_ID=<your-auth0-client-id>
 AUTH0_CLIENT_SECRET=<your-auth0-client-secret>
 ```
 
-Then, run `make testacc`. 
+Then, run `docker-compose up -d && make testacc`. 
 
 **Note:** The acceptance tests make calls to a real Auth0 tenant, and create
-real resources. Certain tests, for example for custom domains
-(`TestAccCustomDomain`), also require a paid Auth0 subscription to be able to
+real resources. Certain tests also require a paid Auth0 subscription to be able to
```

Comment:
Can we list which ones specifically so folks could know to expect those failing tests?